src/Components/Forms/CreateProductForm/CreateProductForm.tsx

```diff
@@ -1,14 +1,41 @@
 import React from 'react';
 import {Formik, Form, Field, FieldInputProps} from 'formik';
+import { useDispatch, useSelector } from 'react-redux';
 import * as Yup from 'yup';
 import FormikInputField from '../../FormikFormControls/FormikInputField/FormikInputField';
 import {FormBlock} from './styled';
+import { addProductStart } from '../../../store/actions/productActions';
+import { IProductState, IProduct } from '../../../types/product';
 
 interface IInput {
   field: FieldInputProps<string>,
 }
 
-export const CreateProductForm = () => (
+const AddProductSchema = Yup.object().shape({
```

Comment:
Please move this to  file 'validation.ts' in this folder

Reply:
Schema moved to validation.ts in this folder